avahi-core/reconfirm.c

```diff
@@ -103,6 +103,8 @@ int avahi_record_reconfirm(
     AvahiProtocol protocol,
     AvahiRecord *r) {
 
+    AvahiRecord *r_assoc;
```

Comment:
The scope of `r_assoc` can be reduced.

Reply:
This seems to be a matter of preference and Avahi doesn't have consistency here. Going to leave as-is unless we can point to some project-policy on this (or that not fixing this is ultimately preventing the feature from merging).